src/Numerics/ODESolvers/StrongStabilityPreservingRungeKuttaMethod.jl

```diff
@@ -149,6 +149,103 @@ end
     end
 end
 
+"""
+    SSPRK22Heuns(f, Q; dt, t0 = 0)
+
+This function returns a [`StrongStabilityPreservingRungeKutta`](@ref) time stepping object
+for explicitly time stepping the differential
+equation given by the right-hand-side function `f` with the state `Q`, i.e.,
+
+```math
+  \\dot{Q} = f(Q, t)
+```
+
+with the required time step size `dt` and optional initial time `t0`.  This
+time stepping object is intended to be passed to the `solve!` command.
+
+This uses the second-order, 2-stage, strong-stability-preserving, Runge--Kutta scheme
+of Shu and Osher (1988) (also known as Heun's method.)
+Exact choice of coefficients from wikipedia page for Heun's method :)
+
+### References
+    @article{shu1988efficient,
+      title={Efficient implementation of essentially non-oscillatory shock-capturing schemes},
+      author={Shu, Chi-Wang and Osher, Stanley},
+      journal={Journal of computational physics},
+      volume={77},
+      number={2},
+      pages={439--471},
+      year={1988},
+      publisher={Elsevier}
+    }
+    @article {Heun1900,
+       title = {Neue Methoden zur approximativen Integration der
+       Differentialgleichungen einer unabh\"{a}ngigen Ver\"{a}nderlichen}
+       author = {Heun, Karl},
+       journal = {Z. Math. Phys},
+       volume = {45},
+       pages = {23--38},
+       year = {1900}
+    }
+"""
+function SSPRK22Heuns(F, Q::AT; dt = 0, t0 = 0) where {AT <: AbstractArray}
+    T = eltype(Q)
+    RT = real(T)
+    RKA = [RT(1)]
+    RKB = [RT(1 // 2), RT(1 // 2)]
```

Comment:
This might work

```suggestion
    RKA = [RT(1) RT(0); RT(1//2) RT(1//2)]
    RKB = [RT(1), RT(1 // 2)]
```

You need to look at the update to make sure you get what you want (since I don't think the Tablueas are in standard form)

https://github.com/CliMA/ClimateMachine.jl/blob/be81d4e972a294ca8431b887b35e5f55cb31879b/src/Numerics/ODESolvers/StrongStabilityPreservingRungeKuttaMethod.jl#L101-L150